scripts/generate-opt-in-redirect-checks.ts

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs'
+import path from 'path'
+import { cachedGetProductData } from 'lib/get-product-data'
+import { productSlugs } from 'lib/products'
+
+const filePath = path.join(
+	process.cwd(),
+	'src',
+	'.generated',
+	'opt-in-redirect-checks.js'
+)
+
+export default async function main() {
+	const result = ['export default {']
+
+	productSlugs.forEach((productSlug) => {
+		if (productSlug === 'hcp' || productSlug === 'terraform') {
```

Comment:
Where are we handling the opt in redirects for these two?

Reply:
In `hashicorp/cloud.hashicorp.com` and `hashicorp/terraform-website` 👍